deepmd/entrypoints/test.py

```
@@ -559,39 +585,50 @@ def test_ener(
+                "rmse_ea": (rmse_ea, energy.size),
+                "rmse_fr": (rmse_fr, force_r.size),
+                "rmse_fm": (rmse_fm, force_m.size),
+                "rmse_v": (rmse_v, virial.size),
+                "rmse_va": (rmse_va, virial.size),
+            },
+            find_energy,
+            find_force,
+            find_virial,
+        )
     if dp.has_hessian:
         dict_to_return["mae_h"] = (mae_h, hessian.size)
```

Ruff (0.8.2)

Undefined name `dict_to_return`

(F821)